4.0.0/upgrade_notes.md

```diff
@@ -17,11 +17,13 @@ Refer to the appropriate workflow, depending on your environment:
 
 For consolidation and to meet higher performance requirements, upgrading to v4.0.0 from v3.x.x involves migrating MySQL to the PostgreSQL database. The migration process is seamless and no user intervention is expected. For migration instructions and troubleshooting tips, see [Migrating MySQL to PostgreSQL Database](./migration.md).
 
-### Elastic search 6.8.6 upgrade
+Please note that Installer will always generate the manifests for `mysql-to-postgresql-migration` and `mysql-latest-migrations` jobs: even if so, the migration will not run again if it has successfully completed once.
```

Comment:
Note that Installer will always generate the manifests for `mysql-to-postgresql-migration` and `mysql-latest-migrations` jobs.  The migration will not run again if it has successfully completed once.